app/pb_kits/playbook/utilities/globalProps.js

```diff
@@ -31,3 +32,14 @@ export const spacing = ({
   css += padding ? `p_${padding} ` : ''
   return css
 }
+
+const darkProps = ({ dark }) => {
+  let css = ''
+  css += dark ? 'dark' : ''
+  return css
+}
```

Comment:
could this be:
`const darkProps = ({ dark }) => {
    return dark ? 'dark' : ''
}`